plugins/inputs/apache/apache.go

```diff
@@ -61,21 +91,53 @@ func (n *Apache) Gather(acc telegraf.Accumulator) error {
 	return outerr
 }
 
-var tr = &http.Transport{
-	ResponseHeaderTimeout: time.Duration(3 * time.Second),
-}
+func (n *Apache) gatherUrl(addr *url.URL, acc telegraf.Accumulator) error {
+	var err error
```

Comment:
Yes you are rigth  , there is a much better way to write this code, I like somebody can teach me about best practices on golang.
But when rewritted code I had had to declare `var tr *http.Transport` . Can you review please if could be a good solution ?